src/main/java/com/work/management/web/rest/controller/EmployeeController.java

```diff
@@ -61,7 +61,8 @@
   @RequestMapping(method = RequestMethod.PUT, value = "/bulkUpdate", produces = "application/json; charset=UTF-8")
   @ApiOperation(value = "Bulk updates on employees details")
   ResponseEntity<List<Employee>> employeeBulkUpdate(
-      @Valid @RequestBody @ApiParam(value = "Bulk update on employee details", required = true) BulkEmployeeRequest bulkEmployeeRequest) {
+      @Valid @RequestBody @ApiParam(value = "Bulk update on employee details", required = true) BulkEmployeeRequest bulkEmployeeRequest)
+      throws Exception {
```

Comment:
Why throws exception is required here?...I dont think its required here, if its required here, list the exact exception this method can throw like BAdrequest